image/php-fpm/www.conf

```diff
@@ -367,13 +367,11 @@ chdir = /
 ;env[TEMP] = /tmp
 
 env[APP_DEBUG] = $APP_DEBUG
-
-env[DB_HOST] = $POSTGRES_PORT_5432_TCP_ADDR
-env[POSTGRES_PORT] = $POSTGRES_PORT_5432_TCP_PORT
-env[DB_DATABASE] = $POSTGRES_DATABASE
-env[DB_USERNAME] = $POSTGRES_USER
-env[DB_PASSWORD] = $POSTGRES_PASSWORD
-
+env[DB_HOST] = $DB_PORT_5432_TCP_ADDR
```

Comment:
I gave an alias of `db` to the `postgres` container. That's why these env var names changed.